readme.md

```diff
@@ -121,6 +121,8 @@ Click the type names for complete docs.
 - [`FixedLengthArray`](source/fixed-length-array.d.ts) - Create a type that represents an array of the given type and length.
 - [`MultidimensionalArray`](source/multidimensional-array.d.ts) - Create a type that represents a multidimensional array of the given type and dimensions.
 - [`MultidimensionalReadonlyArray`](source/multidimensional-readonly-array.d.ts) - Create a type that represents a multidimensional readonly array of the given type and dimensions.
+- [`OneOrMore`](source/n-or-more.d.ts) - Create a type of one or more items.
```

Comment:
I prefer using the word "elements" over "items", so we should use that consistency.

Reply:
Addressed in 446e3c5